src/server/models/db/migrations/20221204202309_scheduler_batch_worker/migration.sql

```diff
@@ -0,0 +1,60 @@
+/*
+  Warnings:
+
+  - The values [ANNUAL_REVIEW_START_ONE_DAY_REMINDER_POST_EMAIL_SENT,ANNUAL_REVIEW_START_ONE_WEEK_REMINDER_POST_EMAIL_SENT,ANNUAL_REVIEW_START_ONE_MONTH_REMINDER_POST_EMAIL_SENT,ANNUAL_REVIEW_STARTED_POST_EMAIL_SENT,ANNUAL_REVIEW_STARTED_PROVIDER_EMAIL_SENT,UNPUBLISHED_POST_EMAIL_SENT,UNPUBLISH_ONE_DAY_REMINDER_POST_EMAIL_SENT,UNPUBLISH_WEEKLY_REMINDER_POST_EMAIL_SENT,UNPUBLISHED_PROVIDER_EMAIL_SENT,UNPUBLISH_ONE_DAY_REMINDER_PROVIDER_EMAIL_SENT,UNPUBLISH_WEEKLY_REMINDER_PROVIDER_EMAIL_SENT] on the enum `AuditEvent` will be removed. If these variants are still used in the database, this will fail.
```

Comment:
This isn't relevant anymore. I'm wondering how this will behave when it's deployed to main (prod db).

I think the best course of action is to 

1. enums are ordered -- so I think it's best to check if any `Audits` are using the "UNDEFINED" type. I would change the schema so "REMINDER" goes at the end for safety. 
2. delete these migration files, start a new psql container, and rerun the migration script. We are deleting/changing things that don't exist on the prod db to begin with